test/test.js

```diff
@@ -249,7 +249,7 @@ describe('anonymous auth', () => {
   beforeEach(() => {
     let count = 0;
     fetchMock.restore();
-    fetchMock.mock(ANON_AUTH_URL, {
+    fetchMock.mock(`begin:${ANON_AUTH_URL}`, {
```

Comment:
just to confirm the reason for this change: does `ANON_AUTH_URL` by itself just not match if the incoming request url has query params attached to it?